components/drawer/events.md

```diff
@@ -14,6 +14,8 @@ This article explains the events available in the Telerik Drawer for Blazor:
 
 * [SelectedItemChanged](#selecteditemchanged)
```

Comment:
When the events are not in a title or heading, use the `monospace` formatting to style them as `code`.
```suggestion
* [`SelectedItemChanged`](#selecteditemchanged)
```